.chronus/changes/typekits-no-magic-program-2025-3-16-16-33-41.md

```diff
@@ -0,0 +1,7 @@
+---
+changeKind: breaking
```

Comment:
we might need to watch putting this because chronus will consider this a major version. Maybe we need a new `breaking-experimental` option?

Reply:
Yeah - I wasn't sure if we even wanted to call out that this is breaking since it is experimental. Probably should mark this as minor for chronus, and then make sure to call out the change in the release notes.